mmv1/products/pubsub/Topic.yaml

```diff
@@ -126,7 +126,7 @@ properties:
           of GCP altogether) will be routed for storage in one of the
           allowed regions. An empty list means that no regions are allowed,
           and is not a valid configuration.
-        required: true
```

Comment:
I'm not seeing how this solves https://github.com/hashicorp/terraform-provider-google/issues/20431

The original debug log shows no required validation message for this field